website/docs/r/netapp_account.html.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_netapp_account"
+sidebar_current: "docs-azurerm-resource-netapp-account"
+description: |-
+  Manage Azure NetApp Account instance.
+---
+
+# azurerm_netapp_account
+
+Manage Azure NetApp Account instance.
+
+
+## NetApp Account Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "acctestRG"
+  location = "Eastus2"
+}
+
+resource "azurerm_netapp_account" "example" {
+  name                = "acctestnetappaccount"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  location            = "${azurerm_resource_group.example.location}"
+
+  active_directories {
+    username            = "aduser"
+    password            = "aduserpwd"
+    smb_server_name     = "SMBSERVER"
+    dns                 = "1.2.3.4"
+    domain              = "westcentralus.com"
+    organizational_unit = "OU=FirstLevel"
+  }
+
+  tags = {
+    env = "test"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the NetApp Account. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The resource group name of the NetApp Account. Changing this forces a new resource to be created.
```

Comment:
Could we word this like the rest of the documentatin?
```suggestion
* `resource_group_name` - (Required) The name of the resource group where the NetApp Account resides. Changing this forces a new resource to be created.
```

Reply:
Fixed